src/views/profile/auth/index.tsx

```diff
@@ -172,6 +173,9 @@ function Auth() {
                 dispatch,
                 user.user
             );
+
+            dispatch(setUserIsBlocked(user.user.blocked));
+            dispatch(setUserIsSuspect(user.user.suspect));
```

Comment:
It must be within `welcomeUser`, so it gets updated both on login and authentication

Reply:
Make sense. Fixed.